lib/core/src/display/rendering/htmlrenderer.rs

```diff
@@ -0,0 +1,63 @@
+use super::Camera;
+use super::HTMLScene;
+use super::IntoCSSMatrix;
+use crate::system::web::StyleSetter;
+use crate::prelude::*;
+
+fn eps(value: f32) -> f32 {
```

Comment:
This definitely needs docs. I assume nobody who has not coded GL engines would understand why its needed.